server/@types/ui/index.d.ts

```diff
@@ -29,6 +29,8 @@ export type YesOrNo = 'yes' | 'no'
 
 export type YesOrNoOrPreferNotToSay = 'yes' | 'no' | 'preferNotToSay'
 
+export type YesNoOrIDK = YesOrNo | 'iDontKnow'
+
```

Comment:
I wonder if the addition of this type -- which is going to be used in several pages -- might be worth its own commit? Or at least a mention in the commit message?

Reply:
yes, good idea. I'll sort that now